test/unit/lookups/olleh_test.rb

```diff
@@ -224,8 +224,20 @@ def test_olleh_route_searching_with_waypoints
     end
   end
 
+  def test_olleh_km2_local_search
+    VCR.use_cassette("address/km2_local_search") do
+      query = Geocoder::Query.new('분당구+689', { places: 5, sr: 'RANK' })
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.